chat-widget/src/components/startchaterrorpanestateful/StartChatErrorPaneStateful.tsx

```diff
@@ -0,0 +1,69 @@
+import { ILiveChatWidgetContext } from "../../contexts/common/ILiveChatWidgetContext";
+import { ILoadingPaneControlProps } from "@microsoft/omnichannel-chat-components/lib/types/components/loadingpane/interfaces/ILoadingPaneControlProps";
+import { ILoadingPaneStyleProps } from "@microsoft/omnichannel-chat-components/lib/types/components/loadingpane/interfaces/ILoadingPaneStyleProps";
+import { LoadingPane } from "@microsoft/omnichannel-chat-components";
+import { findAllFocusableElement } from "../../common/utils";
+import useChatContextStore from "../../hooks/useChatContextStore";
+import { TelemetryHelper } from "../../common/telemetry/TelemetryHelper";
+import { LogLevel, TelemetryEvent } from "../../common/telemetry/TelemetryConstants";
+import { defaultErrorPaneGeneralStyleProps } from "./common/defaultErrorPaneGeneralStyleProps";
+import { defaultErrorPaneTitleStyleProps } from "./common/defaultErrorPaneTitleStyleProps";
+import { defaultErrorPaneSubtitleStyleProps } from "./common/defaultErrorPaneSubtitleStyleProps";
+import { defaultErrorPaneIconStyleProps } from "./common/defaultErrorPaneIconStyleProps";
+import { defaultErrorPaneIconImageStyleProps } from "./common/defaultErrorPaneIconImageProps";
+import { IStartChatErrorPaneProps } from "./interfaces/IStartChatErrorPaneProps";
+import { StartChatErrorPaneConstants } from "../../common/Constants";
+
+export const StartChatErrorPaneStateful = (startChatErrorPaneProps: IStartChatErrorPaneProps) => {
+    const [state, ]: [ILiveChatWidgetContext, Dispatch<ILiveChatWidgetAction>] = useChatContextStore();
+    
+    const generalStyleProps: IStyle = Object.assign({}, defaultErrorPaneGeneralStyleProps, startChatErrorPaneProps?.styleProps?.generalStyleProps);
+    const titleStyleProps: IStyle = Object.assign({}, defaultErrorPaneTitleStyleProps, startChatErrorPaneProps?.styleProps?.titleStyleProps);
+    const subtitleStyleProps: IStyle = Object.assign({}, defaultErrorPaneSubtitleStyleProps, startChatErrorPaneProps?.styleProps?.subtitleStyleProps);
+    const iconStyleProps: IStyle = Object.assign({}, defaultErrorPaneIconStyleProps, startChatErrorPaneProps?.styleProps?.iconStyleProps);
+    const iconImageProps: IImageProps = Object.assign({}, defaultErrorPaneIconImageStyleProps, startChatErrorPaneProps?.styleProps?.iconImageProps);
+
+    const errorUIStyleProps: ILoadingPaneStyleProps = {
+        generalStyleProps: generalStyleProps,
+        titleStyleProps: titleStyleProps,
+        subtitleStyleProps: subtitleStyleProps,
+        iconStyleProps: iconStyleProps,
+        iconImageProps: iconImageProps,
+    };
+
+    const errorPaneTitleText = startChatErrorPaneProps?.controlProps?.titleText ?? StartChatErrorPaneConstants.DefaultStartChatErrorTitleText;
+    const errorPaneSubtitleText = startChatErrorPaneProps?.controlProps?.subtitleText ?? StartChatErrorPaneConstants.DefaultStartChatErrorSubtitleText;
+
+    const errorUIControlProps: ILoadingPaneControlProps = {
```

Comment:
can we confirm if screen readers are describing elements in the pane properly?, or if we need to add aria-labels

Reply:
Screen reader will read the text on the pane.